lib/routes/aiblog-2xv/archives.ts

```diff
@@ -0,0 +1,101 @@
+import { Route } from '@/types';
+import cache from '@/utils/cache';
+import ofetch from '@/utils/ofetch';
+import { parseDate } from '@/utils/parse-date';
+import { load } from 'cheerio';
+
+export const route: Route = {
+    path: '/archives',
+    categories: ['blog'],
+    example: '/aiblog-2xv/archives',
+    parameters: {},
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    radar: [
+        {
+            source: ['aiblog-2xv.pages.dev/archives'],
+            target: '/archives',
+        },
+    ],
+    name: '归档-全部文章',
+    maintainers: ['Liao-Ke'],
+    handler,
+};
+
+async function handler() {
+    const baseUrl = 'https://aiblog-2xv.pages.dev';
+    const response = await ofetch(`${baseUrl}/archives`);
+    const $ = load(response);
+    const list: any[] = [];
+
+    // 遍历每个月份分组
+    $('#top > main > div > div.archive-month').each((_, monthItem) => {
+        // 遍历分组内的每篇文章
+        $(monthItem)
+            .find('.archive-posts .archive-entry')
+            .each((_, postItem) => {
+                const $post = $(postItem);
+                const $link = $post.find('a').first();
+                const $title = $post.find('h3').first();
+                const $dateMeta = $post.find('.archive-meta span');
+
+                list.push({
```

Comment:
Use [map](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Array/map) instead of [push](https://www.measurethat.net/Benchmarks/Show/11532/0/push-vs-map).